subsys/mgmt/smp_bt.c

```diff
@@ -76,9 +76,19 @@ static struct bt_gatt_attr smp_bt_attrs[] = {
 	BT_GATT_CHARACTERISTIC(&smp_bt_chr_uuid.uuid,
 			       BT_GATT_CHRC_WRITE_WITHOUT_RESP |
 			       BT_GATT_CHRC_NOTIFY,
+#ifdef CONFIG_MCUMGR_SMP_BT_AUTHEN
```

Comment:
Would that make sense to have an option to just encrypt or perhaps have a int option where the user can enter the level like we have done with CONFIG_NET_L2_BT_SEC_LEVEL.

Reply:
See my comment here: https://github.com/zephyrproject-rtos/zephyr/pull/20557#issuecomment-552405402